README.md

```diff
@@ -1,13 +1,5 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This repo contains Lujie "Sue" Zhao's portfolio and STEP projects.
 
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+Description related to the intern program can be found [here] (https://buildyourfuture.withgoogle.com/programs/step/)
```

Comment:
I think the space between `[here]` and `(http://...)` is causing the Markdown not to work properly.